Documentation.app/Contents/MacOS/Command-Line.playground/Pages/Generating a Markdown document.xcplaygroundpage/Contents.swift

```diff
@@ -13,27 +13,27 @@ Nef.Playground.needsIndefiniteExecution(false)
 /*:
  ## 🔖 Generating a Markdown document
  Xcode Playgrounds let you write comments in Markdown format using the symbols `//:` for single line comments, or `/•: ... •/` for multiline comments. Inside this comments, you can use any Markdown syntax; an exmaple:
-
+ 
  ```swift
-/•:
-‌‌ ### This is a heading 1
+ /•:
+ ‌‌ ### This is a heading 1
  This is regular text. *This is bold text*. [This is a link](http://bow-swift.io).
```

Comment:
Bold text is 2 asterisks.

```suggestion
 This is regular text. **This is bold text**. [This is a link](http://bow-swift.io).
```